### PR TITLE
Configure queue

### LINKED ISSUE #1194: Make queue system configurable
Currently hard coded to `local`, but we should support `pbs` also.

Note; this requires some small changes in the ensemble evaluator also.

### PR DESCRIPTION
**Pull request by oyvindeide**
**Issue**
Resolves #1194

### CODE REVIEW

File: ert3/config/_ensemble_config.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class _EnsembleConfig(BaseModel):
 
 class ForwardModel(_EnsembleConfig):
     stages: List[str]
-    driver: Literal["local"] = "local"
+    driver: Literal["local", "pbs"] = "local"
```
Comment (lars-petter-hauge) [inline ert3/config/_ensemble_config.py:17]: Do we want to test a config with driver set to `pbs`? e.g. in `test_ensemble_config:test_input`?

There's a balance between testing pydantic of course, but I think adding such a test would test our configuration setup - which makes sense.

**Comment by markusdregi** [comment]
I'm uncertain how valuable it is with the plugin before the API of the evaluator and its concepts are in place. I would prefer if we can separate our the logic that allows `local` or `pbs` in the configuration. Basically the first commit, together with an extended literal that accepts `pbs`. That is a no-brainer to merge and makes testing the `pbs` cluster easier. The second commit I'm not so sure about at this point in time...

**Comment by lars-petter-hauge** [comment]
We had a discussion about that as well, and the commits you're referring to were removed. The reason was that our intention is not to create our own drivers, but to use whatever drivers the queue system we are currently using has to offer. And then it no longer makes sense to allow for other drivers to be installed through our plugin management system (they must be installed via the queue system we are using (currently dask..)).

So we are in agreement :)

File: ert3/config/_ensemble_config.py (hunk 1 above)
Comment (oyvindeide) [reply ert3/config/_ensemble_config.py:17]: Done

**Comment by lars-petter-hauge** [comment]
Otherwise, this PR has been checked out locally and tested as well. Functionality looks good!

**Review by lars-petter-hauge — APPROVED** [review]
LGTM!

**Pull request merged**